src/components/InterfaceEditor.tsx

```diff
@@ -361,10 +361,11 @@ export default ({
   const [mappingToEditIndex, setMappingToEditIndex] = useState(0);
 
   const parseAstarteInterfaceJSON = useCallback(
-    (json: any): AstarteInterface | null => {
+    (json: unknown): AstarteInterface | null => {
       let parsedInterface: AstarteInterface;
       try {
-        parsedInterface = AstarteInterface.fromJSON(json);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        parsedInterface = AstarteInterface.fromJSON(json as any);
```

Comment:
json is already `unknown` is the cast to `any` required?

Reply:
We'll need to check and rework all `fromJSON` methods to accept `unknown` objects and do proper parsing. That's a matter for another PR, I'm leaving the linter warning active for now.